### PR TITLE
Fix Hint and DisplayName not used error when option not selected

### PR DESCRIPTION
**Pull request by qiaoweiz**
resolve: #319

### CODE REVIEW

File: assets/server/popup.html [hunk 1]
```diff
@@ -27,7 +27,7 @@ <h1 class="title">{{ .PageTitle }}</h1>
             {{ if eq $element $context.Category }}
             <option value="{{ $element }}" selected="selected" hint="{{ $value.Hint }}">{{ $value.DisplayName }}</option>
             {{ else }}
-            <option value="{{ $element }}">{{ $element }}</option>
+            <option value="{{ $element }}" hint="{{ $value.Hint }}">{{ $value.DisplayName }}</option>
             {{ end }}
```
[the remark this answers is not in the document]
Comment (qiaoweiz) [reply assets/server/popup.html]: Updated. Thank you.

**Review by sethvargo — APPROVED** [review]

File: assets/server/popup.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (yolocs) [reply assets/server/popup.html]: Is there a reference for `selectedIf`? I was not able to find any doc about its usage

[the remark this answers is not in the document]
Comment (qiaoweiz) [reply assets/server/popup.html]: IIUC, it is a customized function: https://github.com/abcxyz/pkg/blob/main/renderer/funcs.go#L47

[the remark this answers is not in the document]
Comment (yolocs) [reply assets/server/popup.html]: Ah I see. Thanks!

**Review by dandans-verily — APPROVED** [review]

**Pull request merged**